src/main/kotlin/com/coder/toolbox/util/TLS.kt

```diff
@@ -87,7 +90,7 @@ fun coderSocketFactory(settings: ReadOnlyTLSSettings): SSLSocketFactory {
         return sslContext.socketFactory
     }
 
-    return AlternateNameSSLSocketFactory(sslContext.socketFactory, settings.altHostname)
+    return AlternateNameSSLSocketFactory(sslContext.socketFactory, settings.altHostname!!)
```

Comment:
Not a big deal at all but extracting `settings.altHostname` into a local variable would make it smart enough to understand this is not null right?